test/Cruzo1155.ts

```diff
@@ -29,6 +31,7 @@ describe("Testing Cruzo1155 Contract", () => {
   let factory: Contract;
   let token: Contract;
   const serviceFee = 300;
+  const rawVaultFuncSignature = RAW_VAULT_FUNCTION_SIGNATURE
```

Comment:
Meaningless assignment, you can just use the imported RAW_VAULT_FUNCTION_SIGNATURE value directly